Alignment/APEEstimation/python/MisalignmentScenarios_cff.py

```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```

Comment:
the documentation about the `MisalignmentScenario_PhaseI_PseudoAsymptotic`: I believe there is a twiki where the procedure how these numbers were derived is explained.